common/types.go

```diff
@@ -72,6 +72,8 @@ const (
 	UserAgentHeader = "User-Agent"
 
 	BlockCacheRWErrMsg = "Notice: The random write flow using block cache is temporarily blocked due to potential data integrity issues. This is a precautionary measure. \nIf you see this message, contact blobfusedev@microsoft.com or create a GitHub issue. We're working on a fix. More details: https://aka.ms/blobfuse2warnings."
+	POSIXOwnerMeta     = "posix_owner"
```

Comment:
define permissions flags here as well for fns account we will persist the file mode in that.